_posts/2016-04-11-growing-developers-at-greenfield.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "Growing Developers at Greenfield"
+description: "Advice for junior developers getting started in their careers"
+category:
+tags: [lessons, advice, career]
+author: tom_chen
+---
+{% include JB/setup %}
+
+![tree]({{ BASE_PATH
+}}/assets/images/growing-developers-at-greenfield/tree.jpeg)
+
+Before I was a developer at Greenfield, I spent several years working in
+academia. After deciding to make the leap into full-time software development
+and completing [Launch Academy](https://www.launchacademy.com/) last year, I was
```

Comment:
Hmm that's a good point. I was kinda hoping we could get Launch Academy to retweet/reblog this and maybe they'd be more likely to do that if we mention them?